rockpaperscissors2/rockpaperscissors2.js

```diff
@@ -0,0 +1,85 @@
+let choices = ["rock", "paper", "scissors"]
+
+const outcomes = {
+    rock: {loseTo: "paper", winTo: "scissors"},
+    paper: {loseTo: "scissors", winTo: "rock"},
+    scissors: {loseTo: "rock", winTo: "paper"}
+  };
+
+userScore = 0;
+computerScore = 0;
+scoreBoard = document.querySelector(".scoreboard")
+
+
+function getComputerChoice(){
+  int_gen = Math.floor(Math.random()*3);
```

Comment:
https://discuss.codecademy.com/t/how-do-math-random-and-math-floor-work-together/490890/2